js/bkex.js

```diff
@@ -662,19 +692,80 @@ module.exports = class bkex extends Exchange {
         const request = {};
         if (symbols !== undefined) {
             if (!Array.isArray (symbols)) {
-                throw new BadRequest (this.id + ' fetchTickers () symbols argument should be an array');
+                throw new BadRequest (this.id + ' fetchTickers() symbols argument should be an array');
             }
         }
+        let market = undefined;
         if (symbols !== undefined) {
             const marketIds = this.marketIds (symbols);
-            request['symbol'] = marketIds.join (',');
+            const symbol = this.safeString (symbols, 0);
+            market = this.market (symbol);
+            if (market['swap']) {
+                if (Array.isArray (symbols)) {
+                    const symbolsLength = symbols.length;
+                    if (symbolsLength > 1) {
```

Comment:
@Dan-krm so isn't this the same as saying "fetchTickers not supported for swap markets"  if we allow only 1 symbol that's `fetchTicker` implementation, no?

Reply:
@Dan-krm alright